app/components/Onboarding/FormContainer/FormContainer.scss

```diff
@@ -6,11 +6,19 @@
   background: var(--darkestBackground);
 }
 
+.titleBar {
+  height: 20px;
+  width: 100%;
+  position: absolute;
+  -webkit-user-select: none;
+  -webkit-app-region: drag;
+}
+
 .header {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
-  padding: 40px 40px 20px 40px;
+  padding: 20px 40px;
```

Comment:
@mrfelton my onboarding looks a little funky. might wanna keep the `padding: 40px 40px 20px 40px;`

<img width="950" alt="screen shot 2018-09-21 at 11 53 51 am" src="https://user-images.githubusercontent.com/4040039/45894808-17b03200-bd95-11e8-947b-c1f46f04ea4e.png">

Reply:
Good catch. I've updated that @JimmyMow